developer-guides/rest-api/users/setstatus/README.md

```diff
@@ -0,0 +1,38 @@
+# Set Status
+
+Sets a user Status when the status message and state is given.
+
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/users.setStatus` | `yes` | `POST` |
+
+
+## Arguments
+
+| Argument | Example | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `message` | `My status update.` | Required | The user's status message. |
+| `status` | `online` | Optional | The user's status like `online`, `away`, `busy`, `invisible`. |
+
+## Example Call
+
+```bash
+curl -H "X-Auth-Token: 40tB-Cn5YQJ74QMlQXi4Zf4E_-e0P5CrklU2pWOtV9M" \
+     -H "X-User-Id: uunbZHiuEnib8Pawj" \
+     -d '{"message":"My status update", "status": "online"}' \
+     https://graphql.rocket.chat/api/v1/users.setStatus
+```
+
+## Example Result
+
+```json
+{
+    "success": true
+}
+```
+
+## Change Log
+
+| Version | Description |
+| :--- | :--- |
+| `2.1`| Created|
```

Comment:
Also I guess this version is wrong :thinking: as you can see here: https://github.com/RocketChat/Rocket.Chat/pull/13933